codespell_lib/data/dictionary.txt

```diff
@@ -10183,6 +10186,7 @@ domians->domains
 dominaton->domination
 dominent->dominant
 dominiant->dominant
+domonstrate->demonstrate
```

Comment:
Could you add more rows with the -tion, -ting, -tes variants of this typo please.

Reply:
done.